requirements.txt

```diff
@@ -1,3 +1,3 @@
 pymongo==2.7.1
 wheel==0.23.0
-
+numpy==1.9
```

Comment:
I'm very uncomfortable with using numpy here.  This is a lot of over head for a small feature in my opinion.  I have a lot of experience with these packages and I don't see a need to build a dependency on numpy for a base-36-encoding.  Please make sure we remove this package. 

 I'm going to block this from moving forward until an alternate solution is provided.